scripts/core/editor3/helpers/highlights.js

```diff
@@ -353,6 +353,33 @@ export function getHighlightDataAtCurrentPosition(editorState, types) {
     return getHighlightDataAtOffset(editorState, types, selection, 0);
 }
 
+/**
+ * Get list of characters for current selection - handling single block for now
+ *
+ * @param {EditorState} editorState
+ * @returns {List}
+ */
+function getSelectedCharacters(editorState) {
```

Comment:
Maybe you can use this function getDraftCharacterListForSelection.
Better to use startKey, startOffset and lastOffset.